chevah/python-package
# AIX fixes for building on IBM Cloud.

Scope
=====

For latest release, the AIX 7.1 package was built on IBM Cloud, as Eliseni lost network connectivity in our datacenter.


Changes
=======

Do not compile on more than 8 threads on AIX. OpenSSL build was consistently failing with 20 threads, so I've used the number from Eliseni.

Install OpenSSL manually on AIX, as on CentOS 5.

Check for all required commands on AIX, not just the generic ones.


**Drive-by changes**:
  * Re-enabled `compat` tests.
  * Use https://bin.chevah.com:20443/third-party-stuff/ instead of https://binary.chevah.com/third-party-stuff/.
  * Test for the Perl module`Test::Simple`everywhere OpenSSL is built.
  * Updated documented dependencies and their security-related issues.

How to try and test the changes
===============================

reviewers: @adiroiban 

Please review changes.

Try this branch on an IBM cloud VM, e.g. the current one at 169.47.177.73, which has everything setup. Can you please save it? Thanks!

Pine64 doesn't boot for some reason, but otherwise things should be fine.
